doc/mkdocs/docs/api/macros/json_use_implicit_conversions.md

```diff
@@ -0,0 +1,41 @@
+# JSON_USE_IMPLICIT_CONVERSIONS
```

Comment:
Still expecting to remove implicit conversions in the next major version?

Reply:
Yes.

Reply:
Worth mentioning that here?

Reply:
I added notes in 063a288a87b945d9edf2d0f4d5406b69f6a4af92. What do you think?

Reply:
:shipit: